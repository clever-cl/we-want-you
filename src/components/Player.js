import React from 'react'
import '../styles/Player.css'
import Body from './Body'
import Sidebar from './Sidebar'

const Player = ({ spotify, response, data, getResponse }) => {
    return (
        <div className="player">
            <div className="player__body">
                <Sidebar />
                <Body spotify={spotify} response={response} data={data} getResponse={getResponse}/>
            </div>
        </div>
    )
}

export default Player
