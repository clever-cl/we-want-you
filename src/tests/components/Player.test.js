import React from 'react';
import '@testing-library/jest-dom'
import { shallow } from 'enzyme';
import Player from '../../components/Player';

describe('Test in <Header />', () => {
  test('should show component correct', () => {
    const wrapper = shallow(<Player />)
    expect(wrapper).toMatchSnapshot()
  })


})