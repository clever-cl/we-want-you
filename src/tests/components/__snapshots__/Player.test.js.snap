// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test in <Header /> should show component correct 1`] = `
<div
  className="player"
>
  <div
    className="player__body"
  >
    <Sidebar />
    <Body />
  </div>
</div>
`;
